api/v1beta2/ibmpowervscluster_types.go

```diff
@@ -45,13 +43,61 @@ type IBMPowerVSClusterSpec struct {
 	// ControlPlaneEndpoint represents the endpoint used to communicate with the control plane.
 	// +optional
 	ControlPlaneEndpoint capiv1beta1.APIEndpoint `json:"controlPlaneEndpoint"`
+
+	// serviceInstance is the reference to the Power VS service on which the server instance(VM) will be created.
+	// Power VS service is a container for all Power VS instances at a specific geographic region.
+	// serviceInstance can be created via IBM Cloud catalog or CLI.
+	// supported serviceInstance identifier in PowerVSResource are Name and ID and that can be obtained from IBM Cloud UI or IBM Cloud cli.
+	// More detail about Power VS service instance.
+	// https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-creating-power-virtual-server
+	// when omitted system will dynamically create the service instance
+	// +optional
+	ServiceInstance IBMPowerVSResourceReference `json:"serviceInstance"`
+
+	// zone is the name of Power VS zone where the cluster will be created
+	Zone string `json:"zone"`
+
+	// resourceGroup name under which the resources will be created.
+	ResourceGroup string `json:"resourceGroup"`
+
+	// vpc contains information about IBM Cloud VPC resources
+	// +optional
+	VPC VPCResource `json:"vpc,omitempty"`
```

Comment:
Just a nit to keep same pattern, please do change if it is ok
```suggestion
	VPC IBMVPCResourceReference `json:"vpc,omitempty"`
```